### DISCUSSION TITLE
Tutorial Translation

### DISCUSSION THREAD

**Discussion by voidmain443** [networkx/networkx#7701] [General]
I would like to translate nexworkx's tutorial into Korean, but I am wondering if it is possible.

**Comment by Schefflera-Arboricola** [comment]
Thank you, @voidmain443, for asking about this! :)

Currently, there are ongoing discussions about translations within the Scientific Python community. Most of the translation work is coordinated through the [CrowdIn](https://accounts.crowdin.com/scientific-python/login) platform. If you’re considering translating NetworkX's Tutorial page into Korean, you’ll likely need a second person to review the translation. I don't think anyone in the NetworkX core team is fluent in Korean, but you might find people interested in this, in the `translations` channel of the Scientific Python Discord server.

You can find out more here:
- https://github.com/networkx/website/issues/21
- https://blog.scientific-python.org/scientific-python/translations/

And please let us know if you need any help from our side :) 

Thank you :)

**Reply by voidmain443** [reply]
I have team to do the translate is it ok??
